app/assets/javascripts/app.js

```diff
@@ -103,4 +103,11 @@ if (typeof accessibleAutocomplete != 'undefined' && document.querySelector('.aut
       polyfillAutocomplete();
     }
   }
-}
\ No newline at end of file
+}
+
+var continueToYourReturn = document.getElementById('continueToYourReturn');
```

Comment:
so that it acts the same as the back link essentially, taking you back to the page where you clicked the "Come back later" button, I think we implemented this before we did the work to save the continue url in user answers, so if you think it would be better to get it from there rather than having the script we could do that now

Reply:
changed now